### PR TITLE
Rework the job creation flow

### LINKED ISSUE #27: Rework the job flow to include a preview step
landing

job-new

job-preview

job-secret

### PR DESCRIPTION
**Pull request by drapergeek**
- Allow a preview step
- Add links for publishing
- Show the edit/delete options on the show page if the job was looked up by token
  Fixes #27 

screencapture-localhost-4000-jobs-new-1463489746735

screencapture-localhost-4000-jobs-15-1463489854417

screencapture-localhost-4000-jobs-15-1463489871474

screencapture-localhost-4000-jobs-15-1463490256545

### CODE REVIEW

**Comment by kylefiedler** [comment]
Code looks good. Like the script a lot more. Why just applications and web?

**Comment by creuter** [comment]
Yeah, we don't need that close. Removing.

File: web/views/job_view.ex [hunk 1]
```diff
@@ -4,4 +4,29 @@ defmodule Community.JobView do
...
+    job.token == token
+  end
+
+  def trix_input(form, field_name, attrs \\ []) do
+    connected_trix_input(form, field_name) ++
+    trix_tag(field_name, attrs)
+  end
+
+  defp connected_trix_input(form, field_name) do
+    [label(form, field_name),
```
Comment (jsteiner) [inline web/views/job_view.ex]: Any reason you're switching to this syntax?

**Comment by jsteiner** [comment]
One question/comment. Otherwise LGTM

File: web/views/job_view.ex [hunk 2]
```diff
@@ -4,4 +4,29 @@ defmodule Community.JobView do
...
     HtmlSanitizeEx.basic_html(text)
   end
+
+  def can_edit?(job, token) do
+    job.token == token
+  end
+
+  def trix_input(form, field_name, attrs \\ []) do
+    connected_trix_input(form, field_name) ++
+    trix_tag(field_name, attrs)
```
Comment (jsteiner) [inline web/views/job_view.ex]: Maybe indent this, or keep it on one line so it's not lost that you're appending these?

File: web/views/job_view.ex [hunk 3]
```diff
@@ -4,4 +4,29 @@ defmodule Community.JobView do
...
+  end
+
+  defp connected_trix_input(form, field_name) do
+    [label(form, field_name),
+     text_input(form, field_name, id: field_name, style: "display:none")]
+  end
+
+  defp trix_tag(field_name, attrs) do
+    [content_tag(
+     :"trix-editor",
```
Comment (joshuaclayton) [inline web/views/job_view.ex]: Add another space to indent?

File: web/views/job_view.ex (hunk 1 above)
Comment (drapergeek) [inline web/views/job_view.ex]: Moved quickly and didn't look at it. I'm going to fix it now, thanks!

**Pull request merged**